drone-fly-app/src/main/java/com/expediagroup/dataplatform/dronefly/app/context/CommonBeans.java

```diff
@@ -49,6 +51,18 @@ public class CommonBeans {
   @Value("${apiary.listener.list:}")
   private String confListenerList;
 
+  @Value("${apiary.security.protocol:#{null}}")
```

Comment:
do we expect more properties, it would be nice if we can configure this as a map of properties so we can add and change whatever we want.